.env

```diff
@@ -44,3 +44,10 @@ MEME_ENDPOINT=https://meme-api.herokuapp.com/gimme/wholesomememes
 #AZURE_STORAGE_QUEUE_ENDPOINT=https://127.0.0.1:10001/devstoreaccount1
 #AZURE_STORAGE_TABLE_ENDPOINT=http://127.0.0.1:10002/devstoreaccount1
 #AZURITE_ACCOUNT_KEY=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==
+
+# IMAGE PROVIDER
+IMAGE_PROVIDER=Bing
+
+# BING IMAGE SEARCH
+BING_SEARCH_KEY=YOUR-KEY-HERE
```

Comment:
Put in KeyVault

Reply:
I put it into app config - check out how I did that in the updated pr